addons/zeus/functions/fnc_moduleLoadIntoCargo.sqf

```diff
@@ -0,0 +1,61 @@
+/*
+ * Author: 654wak654
+ * Loads the object module is placed on into selected vehicle.
+ *
+ * Arguments:
+ * 0: Module logic <OBJECT>
+ * 1: Synchronized units <ARRAY>
+ * 2: Activated <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC, [bob, kevin], true] call ace_zeus_fnc_moduleLoadIntoCargo
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
```

Comment:
Got popup error:
``` 
Warning Message: Variable '_thing' does not support serialization. Call 'disableSerialization' in the current script (maybe 'CBA_fnc_addBISEventHandler') if you need to use it.
```
Add the following
```
if (canSuspend) exitWith {[FUNC(moduleLoadIntoCargo), _this] call CBA_fnc_directCall;};

if (!(missionNamespace getVariable [QEGVAR(cargo,enable), false])) exitWith {
    [LSTRING(RequiresAddon)] call FUNC(showMessage);
};
```

Reply:
Should I also add it to `moduleSuppressiveFire`, because I used that as a template for this function.

Reply:
already in that one
https://github.com/acemod/ACE3/blob/master/addons/zeus/functions/fnc_moduleSuppressiveFire.sqf#L21

Reply:
Also when the function calls itself through `CBA_fnc_directCall` after logic is deleted, `_logic` parameter will be null for the new call so I'm going to put the first `exitWith` above the deletion.